src/daml/Daml/Trading/Cash/CashRedemption.daml

```diff
@@ -0,0 +1,82 @@
+module Daml.Trading.Cash.CashRedemption where
+
+import Daml.Finance.Interface.Holding.Base qualified as Holding (I)
+import Daml.Trading.Cash.CashTransfer qualified as CashTransfer
+import Daml.Finance.Interface.Types.Common (AccountKey, InstrumentKey(..))
+import Daml.Finance.Interface.Holding.Fungible (ArchiveFungible(ArchiveFungible))
+import Daml.Finance.Interface.Holding.Fungible qualified as Fungible (I, Split(..), SplitResult(..))
+import Daml.Finance.Interface.Holding.Util (getAmount)
+
+template SplitRequest
```

Comment:
may be we should move this to CashTransfer

Reply:
Done

Reply:
Not done